src/Features/Core/Portable/EditAndContinue/EditSession.cs

```diff
@@ -253,8 +253,6 @@ private async Task<ImmutableArray<ActiveStatementExceptionRegions>> GetBaseActiv
 
         internal Solution BaseSolution => _baseSolution;
 
-        private Solution CurrentSolution => _baseSolution.Workspace.CurrentSolution;
```

Comment:
😕 wrong commit?

Reply:
These are frustrating because they have repeatedly appeared in clean-up pull requests, making it impossible to look down the list of commits and know what each one is _actually_ doing.

Reply:
i'm frustrated as well.  i use the git integration in VS and it commonly just misses files and ends up adding them to different commits.  I don't know what to do about this.